app/src/lib/fatal-error.ts

```diff
@@ -19,3 +19,17 @@ export function fatalError(msg: string): never {
 export function assertNever(x: never, message: string): never {
   throw new Error(message)
 }
+
+/**
+ * Unwrap a value that, according to the type system, could be null or
+ * undefined, but which we know is not. If the value _is_ null or undefined,
+ * this will throw. The message should contain the rationale for knowing the
+ * value is defined.
+ */
+export function forceUnwrap<T>(message: string, x: T | null | undefined): T {
+  if (x) {
```

Comment:
Should this check if `x == null`, allowing you to `forceUnwrap('shouldn’t throw', false)`?

Reply:
Probably, good catch!

Reply:
I just added some test to daabb5ee57fb3513b1f2fa1f508caf4835d96d4a so we don't need to eyeball things like this in the future.